x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/alert_context_menu.tsx

```diff
@@ -173,7 +176,14 @@ const AlertContextMenuComponent: React.FC<AlertContextMenuProps & PropsFromRedux
   });
   const { eventFilterActionItems } = useEventFilterAction({
     onAddEventFilterClick: handleOnAddEventFilterClick,
-    disabled: !isEvent || !canCreateEndpointEventFilters,
+    disabled:
+      !isEndpointEvent ||
```

Comment:
Added this `endpoint` check just in case the alert_context_menu gets enabled at some point for events from other integrations and we want to keep this disabled

Reply:
The observations below (on the Alerts page) feel like the expected behavior. Would you be willing to confirm?

- When `agent.type` is `endpoint`, it's possible to add exceptions from the Alerts page, per the screenshot below:

![alerts_endpoint](https://user-images.githubusercontent.com/4459398/167180941-30234380-32e0-451c-bd43-ae78cc95f495.png)

- When `agent.type` is NOT `endpoint`, (`auditbeat` in this example), it's NOT possible to add exceptions AND the tooltip is NOT displayed, per the screenshot below:

![alerts_non_endpoint](https://user-images.githubusercontent.com/4459398/167181145-4a48f973-32da-4920-a090-18f76053610a.png)